.gitignore

```diff
@@ -21,3 +21,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+package-lock.json
```

Comment:
@Nta1e Why did you choose to add the package-lock.json to the .gitignore? It is needed.

Reply:
ok let me remove it

Reply:
@JaymesKat I have implemented the feedback